pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -803,10 +823,11 @@ stdenv.mkDerivation (
       addDriverRunpath
       perl
       pkg-config
-      yasm
     ]
     # Texinfo version 7.1 introduced breaking changes, which older versions of ffmpeg do not handle.
     ++ (if versionOlder version "5" then [ texinfo6 ] else [ texinfo ])
+    # 8.0 is only compatible with nasm, and we don't want to rebuild all older ffmpeg builds at this moment.
+    ++ (if versionOlder version "8.0" then [ yasm ] else [ nasm ])
```

Comment:
Same here. 
Also nasm should only be needed on x86 and would do nothing on other architectures.

Reply:
Not sure if yasm ever did anything on non x86 platforms.

Reply:
I doubt it does